controllers/deactivation/mapper_test.go

```diff
@@ -0,0 +1,90 @@
+package deactivation
+
+import (
+	"testing"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	toolchainv1alpha1 "github.com/codeready-toolchain/api/api/v1alpha1"
+	"github.com/stretchr/testify/require"
+	"sigs.k8s.io/controller-runtime/pkg/handler"
+
+	. "github.com/codeready-toolchain/host-operator/test"
+	"github.com/codeready-toolchain/toolchain-common/pkg/test"
+	"github.com/operator-framework/operator-sdk/pkg/k8sutil"
+
+	"k8s.io/apimachinery/pkg/types"
+)
+
+func TestUserSignupToMasterUserRecordMapper(t *testing.T) {
+	// This is required for the mapper to function
+	restore := test.SetEnvVarAndRestore(t, k8sutil.WatchNamespaceEnvVar, test.HostOperatorNs)
+	defer restore()
```

Comment:
@sbryzak Just curious - do we still need to set this env variable? If not, then feel free to keep it as it is - I'll remove it in my PR. I just wanted to check it with you